tests/rest/media/v1/test_url_preview.py

```diff
@@ -576,11 +577,10 @@ def test_oembed_photo(self):
         }
         oembed_content = json.dumps(result).encode("utf-8")
 
-        end_content = (
-            b"<html><head>"
-            b"<title>Some Title</title>"
-            b'<meta property="og:description" content="hi" />'
-            b"</head></html>"
+        end_content = unhexlify(
+            b"89504e470d0a1a0a0000000d4948445200000001000000010806"
+            b"0000001f15c4890000000a49444154789c63000100000500010d"
+            b"0a2db40000000049454e44ae426082"
```

Comment:
?
I feel as though it was clearer before.
I guess you had a reason to do this?
If sticking with the hex-encoded gobbledygook, please can we have a comment saying what's inside it?

Reply:
Oh actually I guess this is an image with binary content, judging by the null bytes. Mind popping a comment on saying what it is (e.g.: small GIF image, small PNG image, whatever it is?). :)

Reply:
Yeah I copied it from somewhere else in the tests, I should probably make a constant for it and re-use it.

Reply:
See #10834 for factoring this out. I did a separate PR since it seem unrelated enough.